README.md

```diff
@@ -15,10 +15,12 @@
   ~ limitations under the License.
   -->
 
-# What is Uniffle
+# Apache Uniffle (Incubating)
 
-Uniffle is a Remote Shuffle Service, and provides the capability for Apache Spark applications
-to store shuffle data on remote servers.
+Uniffle is a unified remote shuffle service for analytics engines.
```

Comment:
analytics engine?

Reply:
Spark says it's a "unified analytics engine for large-scale data processing", do you have any suggestions?

Reply:
MapReduce seems not to be an analytics engine.